packages/react/src/components/Card/CardMedia/Controls/MediaButton.js

```diff
@@ -5,6 +5,7 @@ const MediaButton = styled('div')`
   backface-visibility: hidden;
   filter: drop-shadow(0 2px 4px rgba(0, 0, 0, 0.3));
   transition: transform ${transition.short};
+  will-change: transform;
```

Comment:
this probably need to be added around all the code! I need to review it to ensure `will-change` is there every time `transition` is defined 🙂